services/movie.py

```diff
@@ -0,0 +1,43 @@
+import init_django_orm  # noqa: F401
+
+
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list = None,

```

Comment:
```suggestion
        genres_ids: Optional[list] = None,
```
Please get familiar with Optional and add this type annotation all over the project where it needed
https://stackoverflow.com/questions/51710037/how-should-i-use-the-optional-type-hint